.github/workflows/validate.yml

```diff
@@ -0,0 +1,82 @@
+name: Test and publish
+
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+    tags: [ v.* ]
```

Comment:
Are we sure this works? Something similar didn't work for Akka gRPC which triggered the change at https://github.com/akka/akka-grpc/commit/bba0d14c539c0f9835d6937ae06e0473c92ebfab , but I don't know whether that was about the list syntax or the tag regex

Reply:
No not yet :)

Reply:
It's not a regex but a glob pattern so it should be `v*`.